src/test/java/neworangehrm/com/opensourcehrm/steps/CommonSteps.java

```diff
@@ -4,58 +4,82 @@
  * Created by Viswa on 12/20/2017.
  */
 
+import cucumber.api.PendingException;
 import cucumber.api.Scenario;
 import cucumber.api.java.After;
 import cucumber.api.java.Before;
 import cucumber.api.java8.En;
 
+import neworangehrm.com.opensourcehrm.pages.CommonWebPage;
+import neworangehrm.com.opensourcehrm.pages.LoginPage;
+import neworangehrm.com.opensourcehrm.pages.UserManagementPage;
+import neworangehrm.com.opensourcehrm.util.ReadTestCases;
 import org.junit.Assert;
 import org.openqa.selenium.OutputType;
 import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebDriverException;
 
 import neworangehrm.com.opensourcehrm.util.DriverUtil;
+import neworangehrm.com.opensourcehrm.util.PageUtil;
 
-import static neworangehrm.com.opensourcehrm.pages.LoginPage.*;
-import static neworangehrm.com.opensourcehrm.util.DataUtil.*;
 import static neworangehrm.com.opensourcehrm.util.DataUtil.baseUrl;
 
 public class CommonSteps implements En {
 
-    DriverUtil driverUtil;
+    DriverUtil driverUtil = new DriverUtil();
+    WebDriver driver = driverUtil.getDriverInstance();
+    PageUtil pageUtil = new PageUtil();
+    ReadTestCases readTestCases;
 
     @Before
     public void setUp() {
 
         driverUtil = new DriverUtil();
     }
     public CommonSteps() {
-        Given("^I am on the (.*) page$", (String webPage) -> {
+        Given("^I am on the \"([^\"]*)\" page$", (String webPage) -> {
             switch(webPage.toLowerCase()){
                 case "login":
                     driverUtil.getDriverInstance().get(baseUrl);
+                    pageUtil.setCurrentPage("login");
                     break;
                 case "dashboard":
-                    driverUtil.getDriverInstance().get(baseUrl);
-                    driverUtil.getDriverInstance().findElement(getElementLocator("username")).sendKeys("admin");
-                    driverUtil.getDriverInstance().findElement(getElementLocator("password")).sendKeys("admin");
-                    driverUtil.getDriverInstance().findElement(getElementLocator("Login")).click();
+                    //driverUtil.getDriverInstance().get(baseUrl);
+                    pageUtil.setCurrentPage("dashboard");
+                    //pageActions.login(driver, "Verify valid admin login");
+//                    driverUtil.getDriverInstance().findElement(getElementLocator("username")).sendKeys("admin");
```

Comment:
please take out all commented code. Commented lines should only be comments on the code.

Reply:
Ok.
Deleted all commented unwanted code, now we can baseline the version as 1.0